lab4d/nnutils/nerf.py

```diff
@@ -445,44 +443,88 @@ def compute_eikonal(self, xyz, inst_id=None, sample_ratio=16):
         else:
             rand_inds = Ellipsis
 
-        g = self.gradient(xyz, inst_id=inst_id)[..., 0]
```

Comment:
Moved computation of `y` outside `self.gradient`, to allow reuse between `compute_eikonal` (called at training time) and `compute_normal` (called at rendering time)